plenum/server/node.py

```diff
@@ -921,6 +921,8 @@ def on_view_change_complete(self):
             for replica in self.replicas.values():
                 replica.clear_requests_and_fix_last_ordered()
         self.monitor.reset()
+        for replica in self.replicas.values():
```

Comment:
Can we call it in `replica.on_view_change_done()` (and call it not for master replica only)?

Reply:
Are you suggest to move on_view_change_done after monitor.reset() and expand it?

```suggestion
        self.monitor.reset()
        for replica in self.replicas.values():
            replica.on_view_change_done()
```